operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -208,8 +208,22 @@ type DseWorkloads struct {
 	SearchEnabled    bool `json:"searchEnabled,omitempty"`
 }
 
+// StorageConfig defines additional storage configurations
+type AdditionalVolumes struct {
+	// Mount path into cassandra container
+	MountPath string `json:"mountPath"`
+	// Name of the pvc
+	// +kubebuilder:validation:Pattern=[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*
```

Comment:
could you explain this pattern in a comment? I don't understand the dot and later part

Reply:
@jimdickinson it's there to validate that we enter a name that can be used to create a resource. I took that one from casskop's code. the first part is supposed to allow to specify a namespace but it's wrong cause PersistentVolumeClaim is a namespaced object. Removing that part. The later part just prevents other characters like colon, underscores etc... It's supposed to avoid creating part of the resources and failing